client/packages/invoices/src/InvoiceService.tsx

```diff
@@ -54,6 +59,7 @@ const InvoiceService: FC = () => {
       <Route path={prescriptionsRoute} element={<PrescriptionListView />} />
       <Route path={prescriptionRoute} element={<PrescriptionDetailView />} />
 
+      <Route path={outboundReturnsRoute} element={<OutboundReturnListView />} />
       <Route path={outboundReturnRoute} element={<ReturnsDetailView />} />
```

Comment:
Should probably keep these names consistent, so make the other one same as yours?

Also -- the imports at the top are separate, can you make it a single import statement?